tests/test_events/test_events_targets.py

```diff
@@ -68,6 +68,94 @@ def test_send_to_cw_log_group():
     message["detail"].should.equal({"key": "value"})
 
 
+@mock_events
+@mock_sqs
```

Comment:
There is a script that tests whether all the required dependencies are included in setup.py, scripts/int_test.sh
The short version is:
For each service: create a venv, install dependencies for that service only, and run the tests for that service

This test will cause the script to fail, as calling mock_sqs will require some imports that haven't been installed for events.

TLDR: Would you mind moving the SQS test to a new file, test_events_integration.py?
In general, all tests that cover multiple services should be declared in a file with that specific name (`test*integration`), so the script knows to exclude those when testing the dependencies

Reply:
I see, wouldn't it be better I just rename this file, because I actually planned to put all kinds of targets in there, one of the next ones will be Lambda and SNS?

Reply:
Sure, that makes sense